src/Entity/User.php

```diff
@@ -4,73 +4,86 @@
 
 use ApiPlatform\Core\Annotation\ApiResource;
 use Doctrine\ORM\Mapping as ORM;
+use Doctrine\ORM\Mapping\DiscriminatorColumn;
+use Doctrine\ORM\Mapping\DiscriminatorMap;
+use Doctrine\ORM\Mapping\InheritanceType;
 
 /**
  * @ApiResource()
  * @ORM\Entity(repositoryClass="App\Repository\UserRepository")
+ * @InheritanceType("JOINED")
```

Comment:
je super kopírovať z netu, ale imho ti chýba pred tie ORM veci slovo `ORM\` :) nemusíš to potom importovať po jednom jak máš vyššie na riadkoch 7-9

Reply:
🤦‍♂️ dík